addons/spreadsheet/static/src/pivot/pivot_table.js

```diff
@@ -93,22 +102,22 @@ export class SpreadsheetPivotTable {
      * @returns {Column[]}
      */
     getMeasureHeaders() {
-        return this._cols[this._cols.length - 1];
+        return this._cols[this.getColHeadersHeight() - 1];
     }
 
     /**
      * Get the number of columns leafs (i.e. the number of the last row of columns)
      * @returns {number}
      */
     getColWidth() {
-        return this._cols[this._cols.length - 1].length;
+        return this._cols[this.getColHeadersHeight() - 1].length;
     }
 
     /**
-     * Get the number of row in each columns
+     * Get the number of row in each column header
      * @return {number}
      */
-    getColHeight() {
+    getColHeadersHeight() {
```

Comment:
Why did you change to `HeadersHeight` ? With this name, it looks like we are getting the height of the header part, while we are getting the number of rows, which is the height of the full column in term of cell

Reply:
Because it _does_ return the height of the header part 🤔 `_cols.length` is the number of rows in the header part of the table

Reply:
Nevermind, when asking yesterday I was think of the header of a spreadsheet, not the header of a pivot, so I didn't undersand why a header could have a different number of row :D

Reply:
What about `getNumberOfHeaderRows` ?